opentelemetry/src/global/logs.rs

```diff
@@ -132,5 +140,6 @@ where
 
 /// Shut down the current global [`LoggerProvider`].
 pub fn shutdown_logger_provider() {
-    let _ = set_logger_provider(NoopLoggerProvider::new());
+    let logger_provider = set_logger_provider(NoopLoggerProvider::new());
+    let _ = ObjectSafeLoggerProvider::shutdown(&logger_provider);
```

Comment:
I think we need to revisit `shutdown_*_provider` methods as they are at the API level. The instrumented libraries shouldn't have the method to shutdown the LoggerProvider enabled by the application.

Reply:
Discussed in SIG meeting and we will address this separate issue